sdk/storage/azure-storage-blobs/vcpkg/CONTROL

```diff
@@ -0,0 +1,7 @@
+Source: azure-storage-blobs-cpp
+Version: @AZ_LIBRARY_VERSION@
+Build-Depends: azure-storage-common-cpp
```

Comment:
This package also depends on azure-core.

Do we need to list bcrypt, LibXml2, ...  and all other dependencies?

Reply:
storage-common depends on azure-core and brings these dependencies, it is not necessary to list them here, I can confirm it all builds when you have nothing installed and request azure-storage-common right away,